.changeset/hungry-wombats-fail.md

```diff
@@ -0,0 +1,9 @@
+---
+"wrangler": patch
+---
+
+feat: text decoder
```

Comment:
Thinking about how this would appear in the changelog. Let's make the first line a sentence that describe the change...

E.g.

```
feat: add `--text` flag to decode `kv:key get` response values as utf8 strings

Previously, all kv values were being rendered directly as bytes to the stdout, which makes sense if the value is a binary blob that you are going to pipe the output into a file, but doesn't make sense if the value is a simple string.
```